### DISCUSSION TITLE
Image Comparison difference picture

### DISCUSSION THREAD

**Discussion by echod13** [ImageMagick/ImageMagick#5404] [Help] [answered]
Hey there, 
I am new to imagemagick and am currently working on a small 2d image comparing script.
When I compare 2 images the output is a blurry picture with the differences highlghted not blurred  in red. Is there an agument to the compare command, which affects the output to only show the differences? LIke e.g., a blank picture with the differing spots in rd

Thanks in advance echod

**Comment by fmw42** [comment, marked as answer]
try

`compare -metric XXX -lowlight-color black image1 image2 result`

see https://imagemagick.org/Usage/compare/#compare

**Reply by echod13** [reply]
Thank you :)
